docs/getting-started/invite-members/README.md

```diff
@@ -80,7 +81,7 @@ Once clicked on **Invite user**, a pop up comes up where you decide to which pro
 
 When you are working in the work package list, you can also invite users from there. For example when you want to set an assignee for a work package that has not got access to OpenProject yet, you can select Invite from the drop down. The same applies to accountables or custom fields for your project.
 
-![Invite-user-assignee](Invite-user-assignee.png)
+![invite-user-work-package-table](Bildschirmfoto 2022-03-02 um 14.29.29.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.